engine/runtime-compiler/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -1012,6 +1012,16 @@ class Compiler(
       case _: Warning => (fansi.Color.Yellow ++ fansi.Bold.On, "warning: ")
       case _          => throw new IllegalStateException("Unexpected diagnostic type")
     }
+
+    def fileLocationFromSection(loc: IdentifiedLocation) = {
```

Comment:
private?

Reply:
It is a local method inside of `processImport`. E.g. it is invisible by default. Can one mark such method private in Scala? Certainly not possible in Java.